winterfell/webapp/src/styles/claimSelectForms.styl

```diff
@@ -1,97 +1,187 @@
-    color: colorWhite
-
-  & .footer-wrapper
-    height: (2 * megaSpacing)
-    padding-top: smallSpacing
-    background-color: colorPrimaryDarkBlue
-
-    & .vfi-button-wrapper
-      width: maxLayoutWidthPx
-      max-width: maxLayoutWidthPct
-      margin: largeSpacing auto
-
-    & .vfi-cancel-btn
+  width: 100%
+  background-color: colorWhite
+
+#select-forms-wrapper
+  width: maxLayoutWidthPct
+  max-width: maxLayoutWidthPx
+  margin: 0 auto
+  padding: largeSpacing 0
+
+  & h3
+    margin-bottom: mediumSpacing
+    color: colorPrimaryBlue
+
+  & h4
+    display: inline-block
+    line-height: 40px
+    color: colorPrimaryBlue
+
+  & .section
+    margin-bottom: largeSpacing
+
+    & .section-header
+      & .progressbar
+        position: relative
+        top: 5px
+        width: 45%
+        height: 25px
+        margin: 0
+        float: right
+
+      & .progress-bar
+        line-height: 25px
+        background-color: colorPrimaryDarkBlue
+
+      & .progress-bar-success
+        background-color: colorLightGreen
+
+  & .form-wrapper
+    padding: smallSpacing 0
+    background-color: colorWhite
+    border-top: 2px solid colorLightGray
+
+    &:last-child
+      border-bottom: 2px solid colorLightGray
+
+    & .form-content
+      display: inline-block
+      width: 70%
+
+      @media screen and (max-width: maxWindowWidthPx)
+        width: 100%
+
+    & .form-buttons
+      display: inline-block
+      width: 30%
+      vertical-align: top
+      text-align: right
+
+      @media screen and (max-width: maxWindowWidthPx)
+        width: 100%
+        text-align: center
+
+      & button
+        display: inline-block
+        width: 150px
+        margin-bottom: smallSpacing
+
+        &.download-btn
+          background-color: colorLightGreen
+
+        @media screen and (max-width: maxWindowWidthPx)
+          display: inline-block
+          margin: 0 smallSpacing smallSpacing
+
+    & .form-title-wrapper
+      display: inline-block
+      vertical-align: middle
+
+      & .title
+        display: inline-block
+        &.required
+          color: colorRed
+
+      & .warning
+        display: inline-block
+        &.required
+          color: colorRed
+
+    & .form-summary
+      margin: smallSpacing 0
+
+    & .state-wrapper
+      display: inline-block
+      position: relative
+      width: 36px
+      height: 36px
+      margin-right: miniSpacing
+      border-radius: 50%
+      background-color: colorLightGray
+      vertical-align: middle
+
+      &.required
+        background-color: colorRed
+
+        & .state-icon
+          transform: translate(-50%, -55%)
+          width: 26px
+          height: 26px
+          background: url('../icons/warning-white.svg')
+          background-repeat: no-repeat
+          background-size: cover
+
+      &.complete
+        background-color: colorLightGreen
+
+        & .state-icon
+          transform: translate(-50%, -50%)
+          width: 22px
```

Comment:
im more ok with the hardcoded icon size, but could be a problem on very small screens, maybe we can think about using media queries or just make them flat out responsive.